src/functions.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful;
+
+/**
+  * Unfortunately PHP has no easy way to create a nice, ISO 8601 formatted date string with milliseconds and Z
+  * as the time zone specifier. Thus this hack.
+  *
+  * @param \DateTimeImmutable $date
+  *
+  * @return string ISO 8601 formatted date
+  */
+ function format_date_for_json(\DateTimeImmutable $date)
```

Comment:
i'm a bit confused by this change. the code looks the same as the old method, but there is also added inconsistency in that there is mixed snake_case and camelCase in the SDK now.